src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -18,50 +18,92 @@ redirects:
+import syntheticConfigureMonitor from 'images/synthetic_screenshot-crop_configure-monitor.webp'
 
-For a look at how to use the Selenium IDE to record your workflows and export them in the format required by New Relic, watch this short YouTube video (3:49 minutes).
+import syntheticSelectLocations from 'images/synthetic_screenshot-crop_select-locations.webp'
+
+
+Scripted browsers emulate a custom user experience by scripting browsers that navigate your website, take specific actions, and ensure specific elements are present. [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/) drives scripted monitors. Each time your script runs, New Relic creates a fully virtualized Selenium-driven Google Chrome browser that navigates your website and follows each script step. Synthetic monitoring includes an IDE-style script editor that suggests functions, locators, and other elements to simplify scripting. See [Scripted browser reference](/docs/synthetics/synthetic-monitoring/scripting-monitors/synthetic-scripted-browser-reference-monitor-versions-chrome-100/) for a list of all available functions.
+
+## How to create a scripted browser [#create-scripted-browser]
+
+1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Synthetic monitoring**</DoNotTranslate>.
+
+2. Click <DoNotTranslate>**Create monitor**</DoNotTranslate>.
+
+3. Select the <DoNotTranslate>**User flow / functionality**</DoNotTranslate> tile to test the availability and functionality of your site in different locations.
+
+    <img
+    title="Create monitor - User flow / functionality"
+    alt="Create monitor - User flow / functionality"
+    src={syntheticUserFlowFunctionality}
+    />
+
+4. Configure the monitor. Fill in the fields you want. Add a name for your monitor.
+
+    <img
+    title="Create monitor - configure monitor"
+    alt="Create monitor - configure monitor"
+    src={syntheticConfigureMonitor}
+    />
+
+5. Click <DoNotTranslate>**Select locations**</DoNotTranslate> to select the location you want to test the monitor.
+
+    <img
+    title="Create monitor - select locations"
+    alt="Create monitor - select locations"
+    src={syntheticSelectLocations}
+    />
+
+6. Click <DoNotTranslate>**Write script**</DoNotTranslate> to generate the script and add the instructions of what you want to browse.
+
+7. Click <DoNotTranslate>**Validate**</DoNotTranslate> if you want to test your script. This may take a few minutes depending on the script. 
+
+8. Click <DoNotTranslate>**Save monitor**</DoNotTranslate>.
 
-<Video
-  id="Y0BCF-MSX44"
-  type="youtube"
-/>
 
 ## Visit a URL [#visiting-a-url]
 
-All scripts begin by specifying which URL the monitor should navigate to. To specify a URL, call `$browser.get("url")`:
+All scripts begin by specifying which URL the monitor should go to. To specify a URL, call `$webDriver.get("url")`:
```

Comment:
So, Navigate is the usual terminology applied to accessing pages in a browser; it's not required though. I'd be inclined to keep it in however, because once a user has reached this part of the documentation, they should be thinking in technical terms. Writing scripts for browser monitors is not a casual endeavour!